src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/depends/sqltypes/arithmetic_test.go

```diff
@@ -1250,3 +1250,14 @@ func TestMax(t *testing.T) {
 		}
 	}
 }
+
+func TestCastToBool(t *testing.T) {
+	tcases := []Value{NewInt64(12), NewInt64(0), NewUint64(1), NewUint64(0), NewFloat64(1), NewFloat64(0), testVal(Decimal, "0"),
```

Comment:
put tcases and  wants in []struct for one-to-one correspondence

Reply:
done